.travis.yml

```diff
@@ -23,25 +23,25 @@ matrix:
         - >-
           NOSEOPTS="--verbose --with-timer" SETUPOPTS=""
           PYLINTOPTS="--rcfile=2.7.pylintrc"
-          PYLINTFILES="azurelinuxagent/daemon azurelinuxagent/agent.py azurelinuxagent/ga setup.py makepkg.py"
+          PYLINTFILES="--ignore=osutil,protocol,utils azurelinuxagent/common/ azurelinuxagent/daemon azurelinuxagent/agent.py azurelinuxagent/ga setup.py makepkg.py"
```

Comment:
Why the ignore?

Reply:
This PR was supposed to be _just_ the loose files in common, not the other folders. Somehow, they made it into the PR, though. I can go ahead and remove the ignore.